core/src/main/java/software/amazon/awssdk/AwsSystemSetting.java

```diff
@@ -148,6 +148,20 @@
     AWS_CONTAINER_CREDENTIALS_RELATIVE_URI("aws.containerCredentialsPath", null),
 
     /**
+     * The full URI path to a localhost metadata service to be used by {@link ContainerCredentialsProvider} when
+     * loading credentials. If this is not specified, credentials will not be automatically loaded from the container.
+     */
+    AWS_CONTAINER_CREDENTIALS_FULL_URI("aws.containerCredentialsFullUri", null),
```

Comment:
There's a lot of properties to do similar things like this. Is it possible to clean them up at all?

Reply:
Are there? Where? There's three as far as I know:

1. an IP for the container-based metadata service with a default of http://169.254.170.2
2. a path to be used for the container-based metadata "AWS_CONTAINER_CREDENTIALS_RELATIVE_URI"
3. a full URI to the local metadata service (AWS_CONTAINER_CREDENTIALS_FULL_URI)

These are actually defined by the containers themselves (eg. ECS, EC2, Greengrass etc)